Week01/Problem02/shashemi_02.py

```diff
@@ -0,0 +1,15 @@
+fibonacci=[1,2]
+k=2
+flag=True
+while flag:
+  if (fibonacci[k-1]+fibonacci[k-2])<4e6:
+    fibonacci.append(fibonacci[k-1]+fibonacci[k-2])
```

Comment:
making very long lists like this can be dangerous. It is not an issue in this case, it's just something to keep in mind!